<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1449</width>
    <height>844</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
       <item>
        <layout class="QGridLayout" name="translateLayout">
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Смещение</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="deltaYSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="4">
          <widget class="QPushButton" name="translatePB">
           <property name="text">
            <string>Выполнить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Перемещение</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="deltaZSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="deltaXSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="rotateLayout">
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="pivRotYSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Опорная точка</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="pivRotXSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="angleZSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Угол</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="pivRotZSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="angleYSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="angleXSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Поворот</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QPushButton" name="rotatePB">
           <property name="text">
            <string>Выполнить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="scaleLayout">
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Масштабирование</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Опорная точка</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="pivScaleXSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="pivScaleYSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="factorZSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="pivScaleZSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="factorYSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="factorXSB">
           <property name="minimum">
            <double>-65536.000000000000000</double>
           </property>
           <property name="maximum">
            <double>65536.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Коэффициент</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QPushButton" name="scalePB">
           <property name="text">
            <string>Выполнить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1449</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="loadMeshAction"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadMeshAction">
   <property name="text">
    <string>Загрузить модель</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Очистить трансформации</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
